include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -108,6 +120,13 @@ def AndOp : UTVariadicRTLOp<"and", [Commutative]>;
 def OrOp  : UTVariadicRTLOp<"or", [Commutative]>;
 def XorOp : UTVariadicRTLOp<"xor", [Commutative]>;
 
+def EQOp   : CmpBinRTLOp<"eq", [Commutative]>;
+def NEQOp  : CmpBinRTLOp<"neq", [Commutative]>;
+def LTOp   : CmpBinRTLOp<"lt">;
+def ULTOp  : CmpBinRTLOp<"ult">;
+def LEQOp  : CmpBinRTLOp<"leq">;
+def ULEQOp : CmpBinRTLOp<"uleq">;
```

Comment:
Out of curiosity, did you consider a model like the [`std.cmpi` instruction](https://mlir.llvm.org/docs/Dialects/Standard/#stdcmpi-cmpiop)?  Being consistent with it would make sense unless there is a reason to deviate.  One diasadvantage of it vs your design is that eq/ne cannot be marked commutative.

One advantage is that lowerCmpOp and some other stuff could be a normal function instead of template.

Reply:
I don't have a strong opinion.  I was guessing that RTL-dialect ops would need to carry a sign flag, which would be easier with different ops.

Reply:
We've managed without a sign flag on other operations this far, so I don't think it is necessary. Lets please start like the standard compare for now - if there is a reason to deviate in the future, we can.  Thanks!